### PR TITLE
[HomeTask4] Adding TestNG and JUnit 5

### PR DESCRIPTION
**Pull request by ArinaNichko**

### CODE REVIEW

File: src/main/java/org/example/cucumber/definitionSteps/MyStepdefs.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class MyStepdefs extends BaseTest {
+    @Given("something")
+    public void something() {
+
+
+        // Navigate to the desired web page
+        driver.get("https://reportportal.epam.com/");
+
+        // Execute JavaScript code to set Authorization header
+        String token = "Zg3XqEDeYWM9pVGiNnThawaBTBbv5bpY_SS0aSFIaR126ykBZXKxc86JfAw-DA9sThbHMUpcMojSoiA0cnXY4iHhz8eMefX1G";
```
Comment (nikon2951) [inline src/main/java/org/example/cucumber/definitionSteps/MyStepdefs.java]: hardcode token in the code bad practice . Please fine another solution

File: src/main/java/org/example/driver/managers/ChromeDriverManager.java [hunk 2]
```diff
@@ -15,6 +15,7 @@ public WebDriver createWebDriver() {
 
         ChromeOptions chromeOptions = new ChromeOptions();
         chromeOptions.setPageLoadStrategy(PageLoadStrategy.NORMAL);
+        chromeOptions.addArguments("--headers=Authorization: Bearer Zg3XqEDeYWM9pVGiNnThawaBTBbv5bpY_SS0aSFIaR126ykBZXKxc86JfAw-DA9sThbHMUpcMojSoiA0cnXY4iHhz8eMefX1G");
```
Comment (nikon2951) [inline src/main/java/org/example/driver/managers/ChromeDriverManager.java]: same here

Comment (ArinaNichko) [reply src/main/java/org/example/driver/managers/ChromeDriverManager.java]: done

File: src/main/java/org/example/cucumber/definitionSteps/MyStepdefs.java (hunk 1 above)
Comment (ArinaNichko) [reply src/main/java/org/example/cucumber/definitionSteps/MyStepdefs.java]: I deleted this logic because now I don't use it

**Review by nikon2951 — APPROVED** [review]

**Pull request merged**